Assets/VRTK/Examples/Resources/Scripts/MoveBlock.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoveBlock : MonoBehaviour
+{
+
+    public float moveYAmount = 20.0f;
+    public float moveSpeed = 1.0f;
+    public float waitTime = 5.0f;
+    public float rotateSpeed = 10.0f;
+
+
+    private float startY;
+    private bool goingUp = true;
+    private float stoppedUntilTime;
+    private float moveUpAmount;
```

Comment:
there seems to be some crazy spacing here, my guess is it's a mix of tabs and spaces.

did you put tabs in front of the protected virtual? if so that's what's causing the crazy spacing :)